packages/perseus/src/widgets.ts

```diff
@@ -302,7 +302,7 @@ export const getRendererPropsForWidgetInfo = (
         // It shouldn't matter what we return here, but for consistency
         // we return the untransformed options, as if the widget did
         // not have a transform defined.
```

Comment:
We should be returning the whole item here "as-is" if we're unable to recognize it. Ideally we should never hit this case.